core/kits/documents/kit.php

```diff
@@ -73,7 +73,25 @@ public function get_css_wrapper_selector() {
 		return '.elementor-kit-' . $this->get_main_id();
 	}
 
+	public function handle_revisions_changed() {
+		add_filter( 'wp_save_post_revision_post_has_changed', function ( $post_has_changed, $last_revision, $post ) {
+			// In case default, didn't determine the changes.
+			if ( ! $post_has_changed ) {
+				$last_revision_document = Plugin::instance()->documents->get( $last_revision->ID );
+				$post_document = Plugin::instance()->documents->get( $post->ID );
+
+				$last_revision_settings = $last_revision_document->get_settings();
+				$post_settings = $post_document->get_settings();
+
+				// Its better to add crc32 signature for each revision and then only compare one part of the checksum.
+				return $last_revision_settings !== $post_settings;
+			}
```

Comment:
A filter should return by default the first argument he received.

Reply:
Handled.